build.sh

```diff
@@ -157,7 +157,9 @@ if [ ! -e "$1" ] ; then
 else
   # parameter 1: pkg-list filename
   export PKGLISTNAME=$1
-  PKGLIST=`grep -v -e "^#" -e "^\s*$" "$1" 2>/dev/null`
+#  PKGLIST=`grep -v -e "^#" -e "^\s*$" "$1" 2>/dev/null`
+  PKGLIST=`perl process_build_artefacts.pl "$1" "$2"`
```

Comment:
Should we be using `$()` instead of backticks at this point?

Reply:
I know this script is littered with backticks throughout, but maybe it's time to "fix" that? That's likely a different PR, this is more conversation than an ask for changes to this PR.

Reply:
Would be good to modernise it, but I agree that's a separate update.  The whole script could probably also be rewritten in Perl but that's a much larger undertaking.  (I suspect the bash script is a historical artefact from early builds where Perl might not have been available or needed to be avoided).